commands/lsp_metals_goto_position.py

```diff
@@ -8,3 +8,9 @@ class LspMetalsGotoPositionCommand(sublime_plugin.WindowCommand):
     def run(self, parameters: List[Any]) -> None:
         if isinstance(parameters, list) and parameters:
             open_location(self.window, parameters[0])
+
+class LspMetalsMetalsGotoLocationCommand(sublime_plugin.WindowCommand):
```

Comment:
The names of classes are automatically translated to string command ids?

Reply:
yeah sublime automatically converts command names from camel case to underscores. 

`LspMetals` is just prefix to avoid collision with other sublime commands from other packages and `MetalsGotoLocationCommand` is the command name from the server